app/javascript/flavours/glitch/features/local_settings/page/index.js

```diff
@@ -406,6 +406,18 @@ class LocalSettingsPage extends React.PureComponent {
           >
             <FormattedMessage id='settings.auto_collapse_media' defaultMessage='Toots with media' />
           </LocalSettingsPageItem>
+          <LocalSettingsPageItem
+            settings={settings}
+            item={['collapsed', 'auto', 'height']}
+            id='mastodon-settings--collapsed-auto-height'
+            placeholder='500'
```

Comment:
```suggestion
            placeholder='400'
```
Shouldn't this be 400 to match the current default?